diff --git a/packages/data-provider/src/request.ts b/packages/data-provider/src/request.ts
--- a/packages/data-provider/src/request.ts
+++ b/packages/data-provider/src/request.ts
@@ -3,6 +3,10 @@
 import * as endpoints from './api-endpoints';
 import { setTokenHeader } from './headers-helpers';
 import type * as t from './types';
+
+if (typeof window !== 'undefined') {
+  axios.defaults.withCredentials = true;
+}
 
 async function _get<T>(url: string, options?: AxiosRequestConfig): Promise<T> {
   const response = await axios.get(url, { ...options });
